lib/rex/proto/kerberos/model/enc_kdc_response.rb

```diff
@@ -4,6 +4,22 @@ module Rex
   module Proto
     module Kerberos
       module Model
+        # Based on https://datatracker.ietf.org/doc/html/rfc6806.html#section-11
+        #    EncKDCRepPart   ::= SEQUENCE {
+        #            key             [0] EncryptionKey,
+        #            last-req        [1] LastReq,
+        #            nonce           [2] UInt32,
+        #            key-expiration  [3] KerberosTime OPTIONAL,
+        #            flags           [4] TicketFlags,
+        #            authtime        [5] KerberosTime,
+        #            starttime       [6] KerberosTime OPTIONAL,
+        #            endtime         [7] KerberosTime,
+        #            renew-till      [8] KerberosTime OPTIONAL,
+        #            srealm          [9] Realm,
+        #            sname           [10] PrincipalName,
+        #            caddr           [11] HostAddresses OPTIONAL
```

Comment:
This isn't decoded yet, I've created a separate ticket for this. I haven't seen the KDC return this value.